src/otx/core/data/entity/visual_prompting.py

```diff
@@ -193,10 +193,14 @@ def pin_memory(self) -> ZeroShotVisualPromptingBatchDataEntity:
         super().pin_memory()
         self.prompts = [
             [tv_tensors.wrap(prompt.pin_memory(), like=prompt) if prompt is not None else prompt for prompt in prompts]
```

Comment:
Is this true? At L159, you declared `prompts: list[list[tv_tensors.TVTensor] | None]`. There seems no change that `prompt` can be `None`.

Reply:
Edited at https://github.com/openvinotoolkit/training_extensions/pull/3090/commits/a261a5dd78908acec69972682afcb6a3211ac524.